common/src/main/java/tachyon/Constants.java

```diff
@@ -114,6 +114,7 @@
   public static final String ZOOKEEPER_ADDRESS = "tachyon.zookeeper.address";
   public static final String ZOOKEEPER_ELECTION_PATH = "tachyon.zookeeper.election.path";
   public static final String ZOOKEEPER_LEADER_PATH = "tachyon.zookeeper.leader.path";
+  public static final String ZOOKEEPER_LEADER_RETRY_COUNT = "tachyon.zookeeper.leader.retry";
```

Comment:
I think somewhere in the parameter name should say "inquire", since this is the number of times to retry while looking for the leader.